sjb/generate.py

```diff
@@ -202,7 +206,7 @@ def parse_action(action):
 elif job_type == "suite":
     registered_names = []
     for child in job_config["children"]:
-        child_config_path = abspath(join(dirname(__file__), "generated", "{}.xml".format(child)))
+        child_config_path = abspath(join(dirname(__file__), "generated", "{0}.{1}".format(child, output_format)))
```

Comment:
Choose a different location or add to .gitignore? We don't want to check these in

Reply:
Added those to gitignore